dist/windows/installer.nsh

```diff
@@ -112,6 +112,13 @@ SectionEnd
 ;--------------------------------
 
 Function .onInit
+  ; create a mutex ensures only one installer is running
+  System::Call 'kernel32::CreateMutexA(i 0, i 0, t "qbt_installer") i .r1 ?e'
+  Pop $R0
+
+  StrCmp $R0 0 +3
```

Comment:
I would suggest using a label instead of an offset.
Besides, `+3` is incorrect.